general-superstaq/general_superstaq/superstaq_client.py

```diff
@@ -193,12 +200,19 @@ def create_job(
             "target": target,
             "shots": int(repetitions),
         }
-
         if method is not None:
             json_dict["method"] = method
+        if kwargs or self._client_kwargs:
+            json_dict["options"] = json.dumps({**kwargs, **self._client_kwargs})
+            if "cq_token" in json_dict["options"]:
```

Comment:
shouldn't this be
```suggestion
            if "cq_token" in kwargs:
```
so it doesn't come up when people do it the when people pass it correctly?

Reply:
Yep. Done.